ansible/TestbedProcessing.py

```diff
@@ -240,6 +243,11 @@ def makeTestbed(data, outfile):
                     ptf = ""
                 if not comment:
                     comment = ""
+                # dut is a list for multi-dut testbed, convert it to string
+                if type(dut) is not str:
+                   dut = dut.__str__()
+                dut = dut.replace(",", ";")
```

Comment:
why is this done?
testbed.csv already has ; separator being used.

Reply:
when its a chassis with multiple LCs each being a dut, the testbed.yaml description looks like this

testbed:
    ardut:                        
        group-name: ardut         
        topo: t2              
        ptf_image_name: docker-ptf
        ptf_ip: 10.250.0.252/16
        ptf: ptf
        server: server_1
        vm_base: VM0100
        dut: "[lc1 , lc2, supervisor]"
        dut_type: chassis
        comment:  Arista chassis Testbed

replacing ',' with ';' is required to keep the duts together under the dut column.

Reply:
ok, is there a reason to have each LC separated with ','? Can it be made similar to multiple duts, which are separated with a ';' ? Is there any other consumer of this data which might be affected if testbed.yaml is changed to have ';' separator for LCs

Reply:
Manding the publishers to always pass duts separate by ';' is fine.
Someone new might still be passing duts separated by ',' and this replace(",", ";") handles both the scenarios